docs/Developers/Writing_Good_Commits.rst

```diff
@@ -103,6 +105,9 @@ Example:
     install it by itself.
 
 -  Maximum of 72 chars excluding newline for *each* line.
+   The recommendation is to do this at 72 characters,
+   so that Git has plenty of room to indent text while still
+   keeping everything under 80 characters overall.
```

Comment:
see https://bugzilla.gnome.org/show_bug.cgi?id=756311#c88 for a more concise and descriptive expanation